src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -20,6 +20,11 @@ public class FindMyRecyclingController {
     @Autowired
     IFacilityService facilityService;
 
+    /**
+     * Returns index
```

Comment:
JavaDoc should describe the method, at an overview level. This JavaDoc isn't necessarily adding any value.